cbrain_task/bids_app_handler/views/_show_params.html.erb

```diff
@@ -0,0 +1,28 @@
+
+Input BidsDataset: <%= link_to_userfile_if_accessible @task.bids_dataset %>
+<p>
+
+<% if @task.params[:_cb_mode].to_s =~ /participant|group/ %>
+
+This is a <strong><%= @task.params[:_cb_mode] %></strong>-stage BidsApp running on the following subjects:
+
+<ul>
+<% @task.selected_participants[0,8].each do |sub| %>
+  <li><%= sub %></li>
+<% end %>
+<% if @task.selected_participants.size > 8 %>
+  <li>... and <%= @task.selected_participants.size - 8 %> more.</li>
+<% end %>
+</ul>
+
+<% elsif @task.params[:_cb_mode] == 'save' %>
+
+This is a final BidsApp task whose job is simply to save the results of the other
```

Comment:
IS this the text for the group analysis, I would change to "This is a final BidsApp task that will be performed over all of the selected participants and saved to the results folder."

Reply:
OKay

Reply:
Wait, no, the final task can be one or the other (just save participants outputs, or run group and save group). The message here is for the mode where it just saves.

Reply:
Addendum to pull request notes: this PR also includes code the for standalone BidsAppExample task that I developed in November. I was working within the same branch and didn't realize that I was joining them together.